### ISSUE TITLE
[Component] Checkbox: Add default accessibility identifier.

### ISSUE THREAD

**Issue by michael-zimmermann** [adevinta/spark#1265]
The CheckBoxUIView doesn't have a default accessibility identifier. 
These are only set, when using the Checkbox group.
Each CheckBox should have a default accessibility identifier.
